rllib/models/catalog.py

```diff
@@ -107,37 +107,38 @@ class ModelCatalog:
     @staticmethod
     @DeveloperAPI
     def get_action_dist(action_space,
-                        config,
+                        config=None,
```

Comment:
Prefer to make arguments for internal APIs mandatory since this prevents common errors like forgetting to pass in the config.